gradio/interface.py

```diff
@@ -109,7 +109,7 @@ def __init__(
         outputs: str | Component | List[str | Component] = None,
         verbose: bool = False,
         examples: Optional[List[Any] | List[List[Any]] | str] = None,
-        cache_examples: bool = False,
+        cache_examples: Optional[bool] = None,
```

Comment:
Same as above, let's update the docstring

Reply:
Wow it cache_examples was missing in the docstring :D